cmd/gen-devnet/rollups.go

```diff
@@ -0,0 +1,74 @@
+// (c) Cartesi and individual authors (see AUTHORS)
+// SPDX-License-Identifier: Apache-2.0 (see LICENSE)
+
+package main
+
+import (
+	"context"
+	"errors"
+	"fmt"
+	"os"
+	"os/exec"
+	"path/filepath"
+
+	"github.com/cartesi/rollups-node/internal/config"
+	cp "github.com/otiai10/copy"
+)
+
+const (
+	PROJECT_NAME = "rollups-contracts"
+	ZIP_FILE     = PROJECT_NAME + ".tar.gz"
+)
+
+// Deploy rollups-contracts to a local anvil instance
+func deployRollupsContracts(ctx context.Context,
+	rollupsContractsPath string) error {
+
+	_, err := os.Stat(filepath.Join(rollupsContractsPath, "hardhat.config.ts"))
+	if errors.Is(err, os.ErrNotExist) {
+		return fmt.Errorf("%s does contain a hardhat project: %v", rollupsContractsPath, err)
+	}
+
+	tmpDir, err := os.MkdirTemp("", "")
+	if err != nil {
+		return fmt.Errorf("failed to create temp dir: %v", err)
+	}
+	defer os.RemoveAll(tmpDir)
+
+	if err := copyContracts(rollupsContractsPath, tmpDir); err != nil {
+		return fmt.Errorf("failed to copy contracts: %v", err)
+	}
+
+	if err := deployContracts(ctx, tmpDir); err != nil {
+		return fmt.Errorf("failed to deploy contracts: %v", err)
+	}
+	return nil
+}
+
+// Copy only the rollups hardhat project
+func copyContracts(srcDir string, destDir string) error {
+	return cp.Copy(srcDir, destDir+string(os.PathSeparator)+"rollups")
```

Comment:
I'm just curious as to why you chose an external dependency over calling `cp` with `exec.Command`. Was it for portability reasons? Because as of now our code isn't exactly portable 😅

Reply:
Yeah, that's rather unnecessary! Thanks for pointing that out. Changed at b90cc13d